### DISCUSSION TITLE
Language bar not showing up for my repository

### DISCUSSION THREAD

**Discussion by aedwardz** [github-linguist/linguist#6752] [Classification]
my repository, Search-Engine is not showing the language bar and I am not sure why.

**Comment by lildude** [comment]
I'm seeing a language bar:

![IMG_0035](https://github.com/github-linguist/linguist/assets/627280/391f695c-d968-419e-a22e-f8b8f517ed2d)

Note, the analysis takes place in the background and can take a while during peak times ([documented](https://github.com/github-linguist/linguist/blob/master/docs/how-linguist-works.md)). It has also been known not to run on the first push to a repo. I see your repo has four commits so I suspect you might have hit this scenario and have since pushed a few more changes which have successfully kicked off the background job.
